File: DataAccess/Entities/Symptom.cs
using DataAccess.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccess.Entities
{
    public class Symptom: BaseEntity
    {
        public required SymptomType SymptomType { get; set; }
        public required SymptomScale SymptomScale { get; set; }
    }
}

File: fodmaplog.client/src/app/domain/Symptom.ts
import { SymptomScale } from "./SymptomScale";
import { SymptomType } from "./SymptomType";

export class Symptom {
    id: number;
    symptomType: SymptomType;
    symptomScale: SymptomScale;
    constructor(id: number, symptomType: SymptomType, symptomScale: SymptomScale) {
        this.id = id;
        this.symptomType = symptomType;
        this.symptomScale = symptomScale;
    }
}